ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -186,6 +186,17 @@ or another sexual and/or gender minority?'>
                             onChange={(value) => this.updateDemographicAttribute('lgbtqIdentity', value)}
                             disabled={!demographicSurvey.identifiesAsLgbtq}/>
       </Section>
+
+      {/*Gender Identity section*/}
+      <Section header='Gender Identity'>
+        <FlexColumn style={{...styles.checkboxAreaContainer, height: '5rem'}}>
+          {AccountCreationOptions.gender.map((gender) => {
+            return this.createOptionCheckbox(gender.label, 'gender',
```

Comment:
Yeah, I think only some of these are checkboxes (combinable) and some (at least "none of these..." should be by themselves.